modules/installation-configuration-parameters.adoc

```diff
@@ -1496,6 +1500,64 @@ Additional Alibaba Cloud configuration parameters are described in the following
 
 endif::alibabacloud[]
 
+ifdef::nutanix[]
+[id="installation-configuration-parameters-additional-vsphere_{context}"]
+== Additional Nutanix configuration parameters
+
+Additional Nutanix configuration parameters are described in the following table:
+
+.Additional Nutanix cluster parameters
+[cols=".^2,.^3a,.^3a",options="header"]
+|====
+|Parameter|Description|Values
+
+|`platform.nutanix.apiVIP`
+|The virtual IP (VIP) address that you configured for control plane API access.
+|IP address
+
+|`platform.nutanix.ingressVIP`
+|The virtual IP (VIP) address that you configured for cluster ingress.
+|IP address
+
+|`platform.nutanix.prismCentral.endpoint.address`
+|The Prism Central domain name or IP address.
+|String
+
+|`platform.nutanix.prismCentral.endpoint.port`
+|The port that is used to log into Prism Central.
+|String
+
+|`platform.nutanix.prismCentral.password`
+|The password for the Prism Central user name.
+|String
+
+|`platform.nutanix.prismCentral.username`
+|The username that is used to log into Prism Central.
+|String
+
+|`platform.nutanix.prismElments.endpoint.address`
+|The Prism Element domain name or IP address. [^1^]
+|String
+
+|`platform.nutanix.prismElments.endpoint.port`
+|The port that is used to log into Prism Element.
+|String
+
+|`platform.nutanix.prismElements.uuid`
+|The universally unique identifier (UUID) for Prism Element.
+|String
+
+|`platform.nutanix.subnetUUIDs`
+|The UUID of the subnet that is linked to the Prism Element, which is managing the {product-title} cluster. [^2^]
```

Comment:
This is a bit confusing. The following is the equiv of the way the network is referenced in vSphere doc... "The Prism Element network that contains the virtual IP addresses and DNS records that you configured."

Reply:
Thank you done.